app/workers/zendesk_ticket_worker.rb

```diff
@@ -1,11 +1,16 @@
 class ZendeskTicketWorker
   include Sidekiq::Worker
-  
+
   def perform(ticket_options)
     if GDS_ZENDESK_CLIENT.users.suspended?(ticket_options["requester"]["email"])
       $statsd.increment("#{::STATSD_PREFIX}.report_a_problem.submission_from_suspended_user")
     else
-      GDS_ZENDESK_CLIENT.ticket.create!(HashWithIndifferentAccess.new(ticket_options))
+      create_ticket(ticket_options) rescue ZendeskAPI::Error::NetworkError
```

Comment:
Maybe we should be logging this, rather than swallowing it inline? I'm guessing `NetworkError` can represent a wider range of failure modes?